connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -222,6 +222,13 @@ private void createWorkerTask() {
         createWorkerTask(TargetState.STARTED);
     }
 
+    private void createWorkerTaskWithErrorToleration() {
```

Comment:
Can we reuse the `createWorkerTask()` method just below by passing a `RetryWithToleranceOperator` argument instead of creating the `WorkerSourceTask` object here?

Reply:
+1 I have refactored the constructors to be cleaner with various parameter lists.